files/models.py

```
@@ -1809,6 +1809,50 @@ def media_url(self):
+    FEATURED = "featured"
+    SAVES = "saves"
+    ACADEMIC = "academic"
+    CURATED = "curated"
+
+    CATEGORY_CHOICES = [
+        (SCREENING, "Screened In"),
+        (FEATURED, "Featured In"),
+        (SAVES, "Saves & Playlists"),
+        (ACADEMIC, "Academic Usage"),
+        (CURATED, "Curated Into"),
+    ]
```

Ruff (0.15.14)

[warning] Mutable default value for class attribute

(RUF012)